### PR TITLE
Setup package

### PR DESCRIPTION
**Pull request by mpsalunggg**

### CODE REVIEW

**Review by vickyadrii — APPROVED** [review]
approved om, mungkin ganti base branch nya jadi development @mpsalunggg

File: .gitignore [hunk 1]
```diff
@@ -24,7 +24,6 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
-yarn.lock
```
Comment (akwanmaroso) [inline .gitignore:27]: no need to exclude yarn.lock
> All yarn. lock files should be checked into source control (e.g. git or mercurial). This allows Yarn to install the same exact dependency tree across all machines, whether it be your coworker's laptop or a CI server. Framework and library authors should also check yarn.

Comment (mpsalunggg) [reply .gitignore:27]: done kaks @akwanmaroso

**Review by akwanmaroso — APPROVED** [review]
LGTM

**Pull request merged**